src/main/java/fi/otavanopisto/kuntaapi/server/discover/EntityUpdater.java

```diff
@@ -108,38 +108,31 @@ public int getTimerInterval() {
       }
     }
     
-    return 1000 * 60;
+    return 1000 * 5;
   }
 
-  private void startTimer(int duration) {
-    TimerConfig timerConfig = new TimerConfig();
-    timerConfig.setPersistent(false);
-    getTimerService().createSingleActionTimer(duration, timerConfig);
-  }
-  
-  @Timeout
-  public void onTimeout() {
-    if (!isStopped()) {
+  private void startTimer(long warmup, long delay) {
+    managedScheduledExecutorService.scheduleWithFixedDelay(() -> {
+      UserTransaction userTransaction = ejbContext.getUserTransaction();
       try {
-        if (isEligibleToRun()) {
+        //userTransaction = lookup();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)